source/MaterialXTest/Runtime.cpp

```diff
@@ -338,13 +338,26 @@ TEST_CASE("Runtime: Nodes", "[runtime]")
     REQUIRE(add2_in1.getSourcePort() == add1_out);
     REQUIRE(add2_in2.getSourcePort() == add1_out);
 
+    // Test node renaming
```

Comment:
Should probably also test the case where you remove foo1 and try to create foo again.